iosxr_iso2vbox.py

```diff
@@ -69,29 +69,54 @@
 import argparse
 from argparse import RawDescriptionHelpFormatter
 import re
+import logging
 
 # Telnet ports used to access IOS XR via socat
 console_port = 65000
 aux_port = 65001
 
 
-def run(command_string, debug=False):
+def run_process_quietly(cmd_string, debug=False):
     '''
-    Execute a CLI command string and return the result.
+    Execute a CLI command in the 'background' as process.
+    '''
+    argv = shlex.split(cmd_string)
```

Comment:
I missed this before - why are you using `shlex.split(cmd_string)` here but `cmd_string.split()` below? Why the inconsistency?